extensions/amp-a4a/0.1/real-time-config-manager.js

```diff
@@ -283,6 +283,21 @@ export function validateRtcConfig_(element) {
     return null;
   }
 
+  // The following loop is to allow vendor macros to be set as objects.
```

Comment:
We should do this at the time we build the macros. Instead what if we modify above macro setup as follows:

validVendorMacros[macro] = rtcConfig['vendors'][vendor][macro];	

becomes:
	         
validVendorMacros[macro] = (isObject(rtcConfig['vendors'][vendor][macro]) ? JSON.stringify : val => val).call(null, rtcConfig['vendors'][vendor][macro]);

This saves a later, redundant iteration